lib/Sema/TypeCheckDecl.cpp

```diff
@@ -2930,7 +2930,7 @@ class DeclChecker : public DeclVisitor<DeclChecker> {
 
     if (!IsSecondPass) {
       for (unsigned i = 0, e = PBD->getNumPatternEntries(); i != e; ++i) {
-        // Type check each VarDecl in that his PatternBinding handles.
+        // Type check each VarDecl in that their PatternBinding handles.
```

Comment:
@Shirk The user @pcbro is definitely trying to get a rise out of someone -- the account is fashioned after the character "PC Principal" in South Park, who's somewhat of a caricature of an politically correct person. This doesn't mean the PRs should be ignored, but the intent behind them is pretty clear to me.